python/ray/serve/tests/test_get_deployment.py

```diff
@@ -0,0 +1,141 @@
+import os
```

Comment:
any reason not to just include this in test_deploy?

Reply:
I like the logical separation but I just discovered that you can group tests in a class; let me do that instead